### PR TITLE
ISDK-2505: Update QuickStarts for 3.0.0

### PR DESCRIPTION
**Pull request by paynerc**
This PR updates the QuickStarts for the 3.0.0 changes.

**Contributing to Twilio**

> All third party contributors acknowledge that any contributions they provide will be made under the same open source license that the open source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: AVPlayerExample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -232,7 +241,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 10.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```
Comment (ceaglest) [inline AVPlayerExample.xcodeproj/project.pbxproj]: 👍

File: AVPlayerExample.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -275,7 +293,7 @@
...
```
Comment (ceaglest) [inline AVPlayerExample.xcodeproj/project.pbxproj]: 👍

File: ObjCVideoQuickstart.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -267,7 +278,7 @@
 				GCC_WARN_UNINITIALIZED_AUTOS = YES_AGGRESSIVE;
 				GCC_WARN_UNUSED_FUNCTION = YES;
 				GCC_WARN_UNUSED_VARIABLE = YES;
-				IPHONEOS_DEPLOYMENT_TARGET = 9.3;
+				IPHONEOS_DEPLOYMENT_TARGET = 10.0;
```
Comment (ceaglest) [inline ObjCVideoQuickstart.xcodeproj/project.pbxproj]: 👍

File: ObjCVideoQuickstart.xcodeproj/project.pbxproj [hunk 4]
```diff
@@ -306,7 +328,7 @@
...
```
Comment (ceaglest) [inline ObjCVideoQuickstart.xcodeproj/project.pbxproj]: 👍

File: README.md [hunk 5]
```diff
@@ -4,7 +4,7 @@
 
 # Twilio Video Quickstart for Objective-C
 
-> NOTE: These sample applications use the Twilio Video 2.x APIs. For examples using our 1.x APIs, please see the [1.x](https://github.com/twilio/video-quickstart-objc/tree/1.x) branch.
+> NOTE: These sample applications use the Twilio Video 3.x APIs. For examples using our 2.x APIs, please see the [2.x](https://github.com/twilio/video-quickstart-objc/tree/2.x) branch. For examples using our 1.x APIs, please see the [1.x](https://github.com/twilio/video-quickstart-objc/tree/1.x) branch.
```
Comment (ceaglest) [inline README.md]: I like.

File: README.md [hunk 6]
```diff
@@ -4,7 +4,7 @@
```
Comment (ceaglest) [inline README.md]: Download: Should we use the 3.x specific download link instead?

Docs: It won't be possible to point to the latest version for the docs ^ use 3.0.0-beta1.

File: Podfile [hunk 7]
```diff
@@ -1,7 +1,7 @@
 source 'https://github.com/CocoaPods/Specs'
 
 workspace 'VideoQuickStart'
-platform :ios, '9.0'
+platform :ios, '10.0'
 
 abstract_target 'TwilioVideo' do
   pod 'TwilioVideo', '~> 2.10'
```
Comment (ceaglest) [inline Podfile]: Use 3.0.0-beta1

Comment (paynerc) [reply Podfile]: Good catch. Missed that here.... Got it in Swift.

Comment (paynerc) [reply Podfile]: Done

**Pull request merged**